azure_billing/scrape.py

```diff
@@ -74,7 +74,7 @@ def data(enrollment, token, month=None):
 
     df = convert_json_df(raw)
 
-    c = Counter('azure_costs', "Costs billed to Azure Enterprise Agreement {}".format(enrollment))
+    c = Counter(metric_name, "Costs billed to Azure Enterprise Agreement {}".format(enrollment))
```

Comment:
see my comment in https://github.com/blue-yonder/azure-cost-mon/pull/4#discussion_r114643510 for the problem with that.